automation/src/test/java/org/greenplum/pxf/automation/features/orc/OrcReadTest.java

```diff
@@ -146,6 +154,12 @@ public void orcReadMultiDimensionalLists() throws Exception {
         runTincTest("pxf.features.orc.read.multidim_list_types.runTest");
     }
 
+    @Test(groups = {"features", "gpdb", "security", "hcfs"})
+    public void orcReadStringsContainingNulByte() throws Exception {
```

Comment:
```suggestion
    public void orcReadStringsContainingNullByte() throws Exception {
```
nit, since we call everything here as `null`